app/models/automated_checks/claim_verifiers/one_login_identity.rb

```diff
@@ -0,0 +1,49 @@
+module AutomatedChecks
+  module ClaimVerifiers
+    class OneLoginIdentity
+      TASK_NAME = "one_login_identity".freeze
+      private_constant :TASK_NAME
+
+      def initialize(claim:, admin_user: nil)
+        self.admin_user = admin_user
```

Comment:
I think for this task there will never be an `admin_user` so we should probably hardcode that.